manticore/core/smtlib/solver.py

```diff
@@ -45,9 +47,14 @@
     "optimize", default=True, description="Use smtlib command optimize to find min/max if available"
 )
 
+consts.add(
+    "solver", default="cvc4", description="Choose default smtlib2 solver (z3, yices, cvc4, race)"
```

Comment:
Are we sure cvc4 should be the default given that we have been using z3 for so long?

Reply:
Yes that should not be cvc4 by default. 
From a quick look yices looks like it should be the default. :) 

* needs to make this an Enum